scripts/ci-entrypoint.sh

```diff
@@ -265,7 +273,10 @@ install_addons() {
         sleep 5
     done
     # install cloud-provider-azure components, if using out-of-tree
-    if [[ -n "${TEST_CCM:-}" ]]; then
+    until get_cloud_provider; do
+        sleep 5
+    done
+    if [[ -z "${IN_TREE:-}" ]]; then
```

Comment:
Do any existing test-infra tests need changing to use this env var (and probably a reference to the new in-tree-specific template)?

Reply:
no, this var is dynamically set via the `get_cloud_provider` func above